apps/envited-x-data-space/common/validator/shacl/shacl.utils.ts

```diff
@@ -15,6 +16,20 @@ export const validateShacl = (shapes: DatasetCore<Quad, Quad>) => async (data: D
   return validator.validate(data)
 }
 
+export const checkIfAssetExists = async (file: File) => {
```

Comment:
Gut feeling tells me this actual api call should only care about the CID, has this CID not been calculated yet?

Reply:
CID hasn't been calculated on the frontend so we need to calculate it